aries-site/src/pages/components/layer/side-drawer-layer.mdx

```diff
@@ -0,0 +1,123 @@
+| **1** | **Close button**  | Closes the layer. | required  | Including the close button in the header in addition to the footer "Cancel" button ensures the user can efficiently exit the layer if desired.                                                                                         |
+| **2** | **Cancel button** | Closes the layer. | required  | The footer actions should be left aligned with primary action on the left and "Cancel" button following. The "Cancel" button should always be present as a complement to the primary action, providing an easy way to leave the layer. |
+
+### When to use double confirmation
+
+When data has been edited or changed and the user tries to close the layer, a double confirmation should be shown. This helps avoid unintentional loss of work.
+
+Learn more about the anatomy and implementation of a [double confirmation](/components/layer#when-to-use-double-confirmation) on the main layer page since this behavior applies to all layer types.
+
+## Scroll behavior
+
+The default behavior of the layer is to allow all of the content to scroll, but this behavior can be modified depending on the layer's contents.
+
+| Layer contents            | Sticky region                                                                                         | Scroll region   | Notes                                                                                                                            |
+| ------------------------- | ----------------------------------------------------------------------------------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| **Filters**               | Footer                                                                                                | Header and body | A sticky footer ensures the user always has access to apply, reset, or cancel out of the filter view and return to the data set. |
+| **Form**                  | None or header can be sticky/appear on scroll up if this added context is valueable for the use case. | Body and footer | The footer actions should be at the bottom of the form to encourage the user to see the entire form before submission.           |
+| **Informational content** | TBD                                                                                                   | TBD             | TBD                                                                                                                              |
+
+### Sticky header
+
+If it would be valuable for the user to maintain the context of the layer's topic, the header can be made persistent or reappear when the user scrolls up.
+
+#### Persistent
```

Comment:
Example here should include a close button and a double confirmation. Also looks like it's allowing me to close the layer by clicking outside of the layer.

Reply:
Yep these are old examples, have to update behavior to align with our guidance.